primitives/beefy/src/witness.rs

```diff
@@ -57,18 +60,18 @@ impl<TBlockNumber, TMerkleRoot> SignedCommitmentWitness<TBlockNumber, TMerkleRoo
 	/// and a merkle root of all signatures.
 	///
 	/// Returns the full list of signatures along with the witness.
-	pub fn from_signed<TMerkelize, TSignature>(
+	pub fn from_signed<TSignatureAggregator, TSignature, TAggregatableSignature>(
```

Comment:
It was supposed to be populated by apk prover after the received enough signatures (see the test) because multiple aggregations can not be aggregated.

Nonetheless, I think we might the commitment here instead.

Reply:
hmm maybe you mean in this function? isn't the return value of `TAggregatableSignature` type?

Reply:
> hmm maybe you mean in this function?

Yes, exactly

> isn't the return value of TAggregatableSignature type?

Doesn't seem to be used for the return type. If I understand correctly, `TAggregatedSignature` is used in the return type. And the code compiles without it.